dist/style.css

```diff
@@ -0,0 +1,3 @@
+span{
```

Comment:
You might want to be a bit more specific with this. All spans in your app will now have a pointer for the cursor. Might be undesired down the road.